test/thermodynamics/heat_transfer/equation_in_homogeneous_medium_in_one_dimension_test.py

```diff
@@ -0,0 +1,26 @@
+from collections import namedtuple
+from pytest import fixture
+from sympy import sin, exp
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.thermodynamics.heat_transfer import (
+    equation_in_homogeneous_medium_in_one_dimension as heat_equation
+)
+
+Args = namedtuple("Args", "temperature chi")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    chi = 1e-5
+    temperature = sin(heat_equation.position) * exp(-chi * heat_equation.time)
```

Comment:
I also suggest to put some actual numbers to temperature function.
You can verify them after `assert expr_equals(lhs, rhs)`